roles/ansible_service_broker/tasks/remove.yml

```diff
@@ -99,11 +99,17 @@
     kind: ConfigMap
 
 # TODO: Is this going to work?
+- shell: >
+    oc get apiservices.apiregistration.k8s.io/v1beta1.servicecatalog.k8s.io -n kube-service-catalog || echo "not found"
+  register: get_apiservices
+  changed_when: no
+
 - name: remove broker object from the catalog
   oc_obj:
     name: ansible-service-broker
     state: absent
-    kind: ServiceBroker
+    kind: ClusterServiceBroker
+  when: not "'not found' in get_apiservices.stdout"
```

Comment:
i'm surprised this isn't being flagged... i would think this should be
`"not 'not found' in get_apiservices.stdout"`